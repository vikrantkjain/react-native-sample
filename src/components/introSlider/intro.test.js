import renderer from 'react-test-renderer';
import IntroSlider from './index';

test('Test intro slider', () => {
  const data = [
    {id: 1, name: 'First'},
    {id: 2, name: 'Second'},
    {id: 3, name: 'Third'},
  ];
  const tree = renderer.create(<IntroSlider data={data} />).toJSON();
  expect(tree).toMatchSnapshot();
});
