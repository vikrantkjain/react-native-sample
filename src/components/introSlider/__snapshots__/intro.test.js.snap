// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test intro slider 1`] = `
<View
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNCViewPager
    collapsable={false}
    initialPage={0}
    layoutDirection="ltr"
    onMoveShouldSetResponderCapture={[Function]}
    onPageScroll={[Function]}
    onPageScrollStateChanged={[Function]}
    onPageSelected={[Function]}
    style={
      {
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <View
      collapsable={false}
      style={
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <View
        style={
          [
            undefined,
            {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
          ]
        }
      />
    </View>
    <View
      collapsable={false}
      style={
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <View
        style={
          [
            undefined,
            {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
          ]
        }
      />
    </View>
    <View
      collapsable={false}
      style={
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <View
        style={
          [
            undefined,
            {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            },
          ]
        }
      />
    </View>
  </RNCViewPager>
  <View
    style={
      [
        {
          "alignSelf": "center",
          "bottom": "12%",
          "flexDirection": "row",
          "height": 40,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          [
            {
              "backgroundColor": "rgba(67, 171, 255, 0.4)",
              "borderRadius": 4,
              "height": 8,
              "margin": 4,
              "width": 8,
            },
            undefined,
          ]
        }
      />
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          [
            {
              "backgroundColor": "#14A7B5",
              "borderRadius": 4,
              "height": 8,
              "margin": 4,
              "width": 8,
            },
            undefined,
          ]
        }
      />
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          [
            {
              "backgroundColor": "rgba(67, 171, 255, 0.4)",
              "borderRadius": 4,
              "height": 8,
              "margin": 4,
              "width": 8,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
</View>
`;
